### ISSUE TITLE
Region/range search is enabled, request for feedback

### ISSUE THREAD

**Issue by nuin** [WormBase/website#5535]
We were able to have region/range search enabled on the newest version and would like some feedback in order to add:

* interesting region(s) as an example
* double check the validity of results
* any other relevant aspect

The new version is on 

http://im-258.wormbase.org/tools/wormmine/genomicRegionSearch.do

and the genomic search link can be accessed fro any page. Please re-assign the issue to me when necessary, and/or assign to anyone that can provide more comments.

Thanks

**Comment by nuin** [comment]
@chris-grove I assigned to you, just because we have been interacting more, please re-assign to anyone else that might have more comments.

**Comment by nuin** [comment]
@chris-grove I restarted the build so things will be back tomorrow, sorry for the inconvenience

**Comment by chris-grove** [comment]
@nuin 
OK, sounds good. Thanks!

**Comment by nuin** [comment]
Good to go. I will have the other issues (and this one) solved in the next build.

**Comment by chris-grove** [comment]
@nuin 
Should I be testing the live site:

http://intermine.wormbase.org/tools/wormmine/begin.do

or on the im-258 site:

http://im-258.wormbase.org/tools/wormmine/begin.do

?

**Comment by nuin** [comment]
Hi Chris

Both should point to the same instance, but please use 

http://intermine.wormbase.org/tools/wormmine/begin.do

@tharris gave some ideas on example regions, but let me know if you have anything else that might be interesting to have there. Changing and deploying is straightforward.

**Comment by chris-grove** [comment]
@nuin 
OK, got it. Thanks!

**Comment by chris-grove** [comment]
@nuin 

While testing the new "Regions" tool (which looks pretty cool, by the way), I noticed that the old gene identifier problem has cropped up again. This was the issue where duplicate genes were being created: one copy has the correct identifiers and the duplicate copy has incorrect identifiers. For example, if I try to create a list with gene ID:

WBGene00018861

I come to the disambiguation page, 

![screen shot 2017-04-12 at 3 20 25 pm](https://cloud.githubusercontent.com/assets/611382/24982374/f499ceb4-1f95-11e7-8b70-ac595e0098b6.png)

showing that there is one gene copy that has the correct identifiers:

primary identifier: WBGene00018861
secondary identifier: F55A4.10
symbol: cutl-21

and there's the other gene copy with incorrect identifiers:

primary identifier: Gene: WBGene00018861
secondary identifier: <blank>
symbol: WBGene00018861

These incorrect gene duplicates are the ones showing up when I do a chromosomal interval search, which is the behavior I experienced earlier before you fixed it in the last version of WormMine. If I remember correctly the problem had to do with gene IDs from the GFF not merging properly.

I can reopen the older ticket or start a new one if you'd like (or leave it here)
